perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -1836,9 +1837,14 @@ private String substituteCommonStrings(Application app, List<ApplicationFormItem
 			String footer = "";
 			// get proper value from attribute
 			try {
-				Attribute attrFooter = attrManager.getAttribute(registrarSession, app.getVo(), URN_VO_MAIL_FOOTER);
-				if (attrFooter != null && attrFooter.getValue() != null) {
-					footer = BeansUtils.attributeValueToString(attrFooter);
+				Attribute attrVoFooter = attrManager.getAttribute(registrarSession, app.getVo(), URN_VO_MAIL_FOOTER);
+				if (attrVoFooter != null && attrVoFooter.getValue() != null) {
```

Comment:
Hi, this imply, that vo mail footer must exists, which is not true. Group footer can be defined independently on vo footer. If it's group application, we should try to fill group attr, if not available we should try vo attr. If it's vo application, we should try to fill vo attr. Thanks.